From db9f69a83b5402daf90fd1d2c44d9e65e37708c5 Mon Sep 17 00:00:00 2001
From: Ronan Abhamon <ronan.abhamon@vates.fr>
Date: Fri, 19 Jan 2024 14:39:17 +0100
Subject: [PATCH 157/180] fix(LinstorSR): add drbd in the blacklist of
 multipath.conf

If DRBD is installed for the first time, and if the multipathd
service is not restarted, volumes can be blocked by this daemon.

Note: the drbd-utils RPM adds the same change in /etc/multipath/conf.d/drbd.conf,
but this package is generally installed using XOSTOR UI and we don't display a
reboot message to be performed like the host update view. So it's probably good
to always have this modification whether we have drbd-utils or not.

Signed-off-by: Ronan Abhamon <ronan.abhamon@vates.fr>
---
 multipath/multipath.conf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/multipath/multipath.conf b/multipath/multipath.conf
index ede40bad..935d3d67 100644
--- a/multipath/multipath.conf
+++ b/multipath/multipath.conf
@@ -23,6 +23,7 @@ blacklist {
 	devnode "scini*"
 	devnode "^rbd[0-9]*"
 	devnode "^nbd[0-9]*"
+	devnode "^drbd[0-9]*"
 }
 # Leave this section in place even if empty
 blacklist_exceptions {
